### ISSUE TITLE
Cutting and pasting rich text with pictures: Pictures get removed after submission

### ISSUE THREAD

**Issue by narasimhakumar** [18F/nsf-sbir#1447]
## Expected Behavior
Adding rich text (rtf) with pictures to the seedfund application's "boxes" should work.

## Actual Behavior
The pictures were removed once the application has been submitted.

## Steps to reproduce the behavior
- [Create a text file with pictures using Microsoft Word ]
- [Cut and paste the page into one of the cells of the seed-fund application ]
- [ ]

## This issue is done when:
- [ After submission, when NSF sends the confirmation email with the link to the submission]
- [ ]
- [ ]

**Comment by kmonterr** [comment]
Hi @narasimhakumar - Are you referring to the Project Pitch submission?

**Comment by narasimhakumar** [comment]
Yes.  How could one cut and paste jpeg (or other pictures)? regards
